.jshintrc

```diff
@@ -23,7 +23,7 @@
   "undef"         : true,     // true: Require all non-global variables to be declared (prevents global leaks)
   "unused"        : false,    // true: Require all defined variables be used
   "strict"        : false,    // true: Requires all functions run in ES5 Strict Mode
-  "maxparams"     : 10,       // {int} Max number of formal params allowed per function
+  "maxparams"     : 12,       // {int} Max number of formal params allowed per function
```

Comment:
@AparnaKarve  @chriskacerguis  @AllenBW I disabled that test as I cannot to my knowledge remove anything from `function StateController(...)`. @AparnaKarve was right that bending it over and over again makes it pointless anyway. If there is a nicer way to work around that test, please let me know :)